pkgconfig/meson.build

```diff
@@ -0,0 +1,29 @@
+pkgconf = configuration_data()
+
+pkgconf.set('prefix', get_option('prefix'))
+pkgconf.set('exec_prefix', '${prefix}')
+pkgconf.set('libdir', '${prefix}/@0@'.format(get_option('libdir')))
+pkgconf.set('includedir', '${prefix}/@0@'.format(get_option('includedir')))
+pkgconf.set('LIBVA_VERSION', libva_version)
+pkgconf.set('VA_API_VERSION', va_api_version)
+pkgconf.set('LIBVA_DRIVERS_PATH', driverdir)
+
+pkg_install_dir = '@0@/pkgconfig'.format(get_option('libdir'))
+
+pkg_files = [
+  'libva',
+  'libva-drm',
+  'libva-x11',
+  'libva-glx',
+  'libva-wayland'
+]
```

Comment:
need to honor the meson_options... i.e. don't install pc file if component is disabled.